scripts/commit.sh

```
@@ -887,6 +888,7 @@ function commit_script {
     # Use an editor and commitmsg file
     if [ -n "$msg" ]; then
         commitmsg_file=$(mktemp "/tmp/commitmsg.XXXXXX")
+        trap "rm -f '$commitmsg_file'" EXIT INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)